docs.feldera.com/docs/operations/guide.md

```diff
@@ -125,28 +151,41 @@ making them eviction candidates. To raise their priority:
 
 **Solution**: Ensure the compiler-server has sufficient disk space (20Gib by default, configured via the `compilerPvcStorageSize` value in the Helm chart).
 
-## Diagnosing Performance Issues
-
-When investigating pipeline performance, Feldera support will typically request a support-bundle.  The bundle can be downloaded from your installation with the following:
-
-* The `support-bundle` [fda command](/interface/cli):
-
- ```bash
- fda support-bundle affected-pipeline-name
- ```
-* the `support_bundle` [function in the Python SDK](https://docs.feldera.com/python/examples.html#retrieve-a-support-bundle-for-a-pipeline).
-* or the `support_bundle` [endpoint in the REST API](/api/download-support-bundle).
-
-The support bundle contains the following content:
-
-1. **Pipeline Logs**: for warnings and errors from the [logs](/api/stream-pipeline-logs) endpoint.
-
-2. **Pipeline Configuration**: the [pipeline configuration](/api/get-pipeline), including the SQL code and connector settings.
-
-3. **Pipeline Metrics**: from the [pipeline metrics](/api/get-pipeline-metrics) endpoint.
-
-3. **Endpoint Stats**: from the [stats](/api/get-pipeline-stats) endpoint.
-
-4. **Circuit Profile**: from the [circuit profile](/api/get-performance-profile) endpoint.
-
-5. **Heap Profile**: from [heap usage](/api/get-heap-profile) endpoint.
+## Uncommon Problems
+
+### Lost or accidentally deleted the Feldera PostgreSQL database
+
+Feldera tracks state about pipelines inside a PostgreSQL database. If for
+some reason the state in this database is ever lost or otherwise can not be
+recovered, and the feldera instance had running pipelines at the time, a
+manual intervention may be necessary to clean-up the leftover pods.
```

Comment:
make it clear you are not recovering the pipelines

Reply:
```
... it's not possible to "reassociate" these leftover (orphaned) pipelines.
In this case, the leftover Kubernetes objects backing these pipelines
must be manually removed.
```